.eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     },
   },
   rules: {
+  	"no-bitwise": "off",
```

Comment:
nit: indentation

Reply:
Will use `lint:fix` to auto fix these erros by merging the master brach